app/helpers/view_helper.rb

```diff
@@ -15,4 +15,15 @@ def badge_by_state(state)
     end
   end
 
+  def badge_by_role(role)
+    case role
+      when 'admin'
+        'badge-success'
+      when 'compliance'
+        'badge-warning'
+      when 'member'
+        'badge-info'
+    end
```

Comment:
It's always a good idea to catch `no role`, `new role` with `else` close in `case` statement, imo

Reply:
@mitjok do you mean I need add 'else' condition?